examples/transformerconfigs/README.md

```diff
@@ -22,11 +22,14 @@ create: false
 ```
 If `create` is set to true, it indicates the transformer to create the path if it is not found in the resources. This is most useful for label and annotation transformers, where the path for labels or annotations may not be set before the transformation.
 
-## prefix transformer
-Name prefix transformer adds prefix to the `metadata/name` field for all resources with following configuration:
+## prefix/suffix transformer
+Name prefix suffix transformer adds prefix and suffix to the `metadata/name` field for all resources with following configuration:
 ```
 namePrefix:
 - path: metadata/name
+
+nameSuffix:
+- path: metadata/name
```

Comment:
Can you remove the lines since the transformer only reads from configs for `namePrefix`.
```
nameSuffix:
- path: metadata/name
```

Reply:
Sorry for my late reply. Fixed it.